.github/workflows/stale.yml

```diff
@@ -21,10 +21,13 @@ jobs:
     - uses: actions/stale@v7
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
-        days-before-stale: 60
+        days-before-issue-stale: 30
+        days-before-pr-stale: 10
         days-before-close: 7
```

Comment:
This will auto-close PRs after 17 days of inactivity, but per our current policies it should be 30 days so we should also update the policy:

https://eslint.org/docs/latest/maintain/review-pull-requests#when-to-close-a-pull-request

Reply:
Actually, since there's `exempt-pr-labels: accepted`, this applies to not-accepted PRs only, i.e. PRs we're still evaluating, so perhaps the settings should be the same as for issues? Or, if the intent is to notice staled accepted PRs, we should remove the `exempt-pr-labels: accepted` line.

Reply:
Good call. Updating. I really just don't want stale PRs sitting around without any notice. The chances that someone will come back after 90 days of inactivity to finish up is pretty small.